packages/web-components/.env.example

```diff
@@ -26,3 +26,4 @@ KALTURA_UICONF_ID=<Kaltura uiconf id, e.g. 12905712>
 # Feature Flags
 DDS_FLAGS_ALL=<Boolean flag to turn on all feature flags>
 DDS_CALLOUT_DATA=<Boolean flag to turn on/off callout data>
+DDS_PROMO_GROUP=<Boolean flag to turn on/off callout data>
```

Comment:
@proeung minor thing, the description should be for promo group and not callout data